test/scenario/video_stream.cc

```diff
@@ -204,6 +204,20 @@ CreateH264SpecificSettings(VideoStreamConfig config) {
   return nullptr;
 }
 
+#ifdef WEBRTC_USE_H265
+rtc::scoped_refptr<VideoEncoderConfig::EncoderSpecificSettings>
+CreateH265SpecificSettings(VideoStreamConfig config) {
+  RTC_DCHECK_EQ(config.encoder.layers.temporal, 1);
+  RTC_DCHECK_EQ(config.encoder.layers.spatial, 1);
+
+  VideoCodecH265 h265_settings = VideoEncoder::GetDefaultH265Settings();
+  h265_settings.frameDroppingOn = config.encoder.frame_dropping;
+  h265_settings.keyFrameInterval =
+      config.encoder.key_frame_interval.value_or(0);
+  return new rtc::RefCountedObject<
+      VideoEncoderConfig::H264EncoderSpecificSettings>(h264_settings);
+}
+
```

Comment:
Where is the paring #endif?

Reply:
It looks like it's duplicated during rebasing. I've removed the duplication, but I think the code still doesn't compile. Since it's part of tests, let's fix it when we enable tests.